R/length.R

```diff
@@ -1,18 +1,49 @@
 #' Assign SAS Length
 #'
-#' Assigns SAS length from a variable level metadata to a given data frame.
+#' Assigns SAS length from a variable level metadata to a given data frame. If a
```

Comment:
```suggestion
#' Assigns SAS length from a metadata object to a given data frame. If a
```

variable level metadata (VLM) is something used in dataset programming and I think users might get confused.  I think all the variable level metadata references should be replaced with metadata or metadata object